manifests/run.pp

```diff
@@ -163,6 +163,14 @@
 
   if $restart_service {
     File[$initscript] ~> Service["docker-${sanitised_title}"]
+
+    exec { "check-whether-image-id-changed-${sanitised_title}":
+      command => '/bin/true',
+      onlyif  => "test \"$(service docker-${sanitised_title} imageid)\" != \"$(docker inspect -f \'{{.Id}}\' $image)\"",
```

Comment:
docker maybe available elsewhere, you can see the use of `docker_command` elsewhere in the module.